controllers/resource/testdata/cloudstackKubeadmconfigTemplateSpec.yaml

```diff
@@ -14,6 +14,7 @@ spec:
             anonymous-auth: "false"
             provider-id: cloudstack:///'{{ ds.meta_data.instance_id }}'
             read-only-port: "0"
+            node-labels: cluster.x-k8s.io/failure-domain='{{ds.meta_data.availability_zone}}'
```

Comment:
should we put a hold on this PR, if an updated CAPC version is a prerequisite?

Reply:
This one does not depends on CAPC since `ds.meta_data.availability_zone` is a known/existed cloudstack metadata